cmake/HelperBoost.cmake

```diff
@@ -4,6 +4,7 @@
 # The custom path will be at `C:\\local\`
 if(WIN32)
     list(APPEND BOOST_ROOT "C:/local")
+    list(APPEND BOOST_ROOT $ENV{BOOST_ROOT_1_72_0})
```

Comment:
Perhaps this is just experimentation, but isn't it dangerous to tie ourselves to this particular version (1.72.0)?

Reply:
Well, we only have 2 versions of boost in the pipeline [see](https://github.com/actions/virtual-environments/issues/661).

It's necessary to chose one or the other. Or, another option would be to add a shell scripts which can download latest version of boost and put it in a specific folder.

At the moment I could resolve the initial issue, but now the compilation cannot find the boost header files ...
It's really hard to debug like that, if you have any suggestion.

Reply:
My point is about these lines being not tied to our Azure set-up. They should work outside of our CI, and hence with arbitrary versions, or am I missing something?

Reply:
I am looking at the log but I can only see a massive amount of warnings and no error in the compilation itself. Did I miss the error?

Reply:
Look for `fatal`

Reply:
Okay, printing the content of `boost` folder on the pipeline shows that headers are missing (like `range.hpp`).

I'll ask directly the maintainers of the azure pipelines is there's an issue here or we're missing something.

Reply:
Just out of curiousity, was there a problem from azure or was it something else?